src/Microsoft.DocAsCode.MarkdigEngine.Extensions/MarkdownExtensions.cs

```diff
@@ -29,6 +29,7 @@ public static MarkdownPipelineBuilder UseDocfxExtensions(this MarkdownPipelineBu
                 .UseXref()
                 .UseEmojiAndSmiley(false)
                 .UseTabGroup(context)
+				.UseChromelessForm(context)
```

Comment:
nit: is this because template prefer to uses tabs and docfx prefer to use spaces?